### DISCUSSION TITLE
option to have flashlight emulate stable flashlight

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20553] [Ideas] [answered]
I like using lazer to practice flashlight as I can change the diameter in game, instead of needing a skin, but since the lazer flashlight is completely different from the stable flashlight, it is hard to get a proper feel for how a section would actually look without going into stable and FCing up to that part; it would be nice if there were an option to emulate the fade and sizes of stable, maybe triggered by the classic mod?

**Comment by smoogipoo** [comment]
> but since the lazer flashlight is completely different from the stable flashlight

Citation needed. It's supposed to match, so can you show examples of how it's different?

**Reply by Doksha** [reply]
there is barely any fade (it looks more like bad anti-aliasing) and the starting diameter is much smaller
I thought it was intended to be an entire rework of the mod with how much is changed, follow delay wasnt even added for a while

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/20559
